### PR TITLE
ZER-63 Move IN function usage from RangeField to Calculation

### PR DESCRIPTION
**Pull request by ochupa**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by PavloMS — APPROVED** [review]

File: app/models/calculation_resolver.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    generate_since_function(calculator)
+    generate_from_list_function(calculator)
+    calculator.evaluate(value, parameters)
+  end
+
+  def self.generate_calculator
+    Dentaku::Calculator.new
+  end
+
+  def self.generate_since_function(calculator)
```
Comment (maksymst) [inline app/models/calculation_resolver.rb:17]: What is the purpose of this method?

File: app/models/calculation_resolver.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    calculator.add_function(:since, :numeric, Since.calculate_units)
+  end
+
+  def self.generate_from_list_function(calculator)
```
Comment (maksymst) [inline app/models/calculation_resolver.rb:21]: same as above

File: app/models/calculation_resolver.rb [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  require 'functions/from_list'
+
+  def self.result(parameters, value)
+    calculator = generate_calculator
+    generate_since_function(calculator)
+    generate_from_list_function(calculator)
+    calculator.evaluate(value, parameters)
+  end
+
+  def self.generate_calculator
```
Comment (maksymst) [inline app/models/calculation_resolver.rb:13]: do you really generate anything?

**Review by maksymst — CHANGES_REQUESTED** [review]

File: app/models/calculator_resolver.rb [hunk 4]
```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require 'functions/since'
+
```
Comment (maksymst) [inline app/models/calculator_resolver.rb:4]: do you need this?

**Review by maksymst — CHANGES_REQUESTED** [review]

File: app/models/calculator_resolver.rb (hunk 4 above)
Comment (ochupa) [reply app/models/calculator_resolver.rb]: No, I don't :-) 
Deleted.
Thanks.

**Review by maksymst — APPROVED** [review]

**Review by aboriiisova — APPROVED** [review]

**Pull request merged**